notes-app-client/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "notes-app-client",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
I think these changes to the package-lock are not necessary, there's no package changes in your PR